django/contrib/gis/db/backends/oracle/introspection.py

```diff
@@ -24,12 +21,11 @@ def get_geometry_type(self, table_name, geo_col):
                     (table_name.upper(), geo_col.upper())
                 )
                 row = cursor.fetchone()
-            except Exception as msg:
-                new_msg = (
+            except Exception:
+                raise Exception(
```

Comment:
I think so (although some of the formatting improvements are welcome). A second opinion from someone who worked more with the new style wouldn't hurt.